k8s/bases/swe-idcc/swe-idcc-app.yaml

```diff
@@ -86,7 +86,7 @@ metadata:
 data:
   process-metadata.json: |-
     {
-      "processName": "SWE IDCC",
+      "processName": "SWE IDCC DACF",
```

Comment:
I think that we need also to rename the process in apps-metadata-server-configmap.yaml to be coherent. We can discuss with the PO if we should also to change the URL

Reply:
YEs I agree but it was absent from the file since last commit for deployment / release